### PR TITLE
benchmarking

### PR DESCRIPTION
**Pull request by JGiter**
Benchmark initialization

### Contributor checklist

- [x] Breaking changes - check for any existing interfaces changes that are not backward compatible, removed method etc.
- [x] Documentation - document your code, add comments for method, remember to check if auto generated docs were updated.
- [x] Tests - add new or updated existed test for changes you made.
- [x] Migration guide - add migration guide for every breaking change.
- [x] Configuration correctness - check that any configuration changes are correct ex. default URLs, chain ids, smart contract verification on [Volta explorer](https://volta-explorer.energyweb.org/) or [EWC explorer](https://explorer.energyweb.org/).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,5 @@ compiled
 ethers
 site
 ethers-staking
+*.heapprofile
+*.cpuprofile
```
Comment (jrhender) [inline .gitignore:16]: Are these files being created by `runner.js` I didn't notice that this was the case.

File: benchmarks/runner.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+const { execFile } = require('child_process');
```
Comment (jrhender) [inline benchmarks/runner.js:1]:
```suggestion
/*
 * This script needs to be run on linux, due to use of `top` command
 */

const { execFile } = require('child_process');
```

**Review by jrhender — APPROVED** [review]
I think we can merge it in. Seems like a useful benchmarking script to have around.

File: .gitignore (hunk 1 above)
Comment (JGiter) [reply .gitignore:16]: They are created by Node.js profiler. We don't need them now. I will remove them

**Pull request merged**